server/config.js

```diff
@@ -6,7 +6,8 @@ const cache = new Map()
 const defaultConfig = {
   webpack: null,
   poweredByHeader: true,
-  distDir: '.next'
+  distDir: '.next',
+  assetPrefix: ''
```

Comment:
@elgubenis we don't wanna go that path.
We just wanna support typical CDNs and most of them as origin support and we don't want to re-write any URLs.